### ISSUE TITLE
JAWS doesn't announce presence of ARIA sub menu on down arrow

### ISSUE THREAD

**Issue by mraccess77** [FreedomScientific/VFO-standards-support#377]
### Summary
JAWS does not announce the presence of a sub menu when down arrowing in forms mode on an ARIA menu.

### STR
1. go to https://www.w3.org/TR/wai-aria-practices-1.1/examples/menubar/menubar-1/menubar-1.html#
2. Tab to menu bar
3 Arrow right through items to admissions
4 Down arrow through to tuition sub menu

### Expected result
Submenu should be announced.

### Actual result
Submenu is not announced

#### JAWS version and build number
JAWS 2020
 
#### Operating System and version
Windows 10

#### Browser and version:
Chrome 80.80 and Firefox 74

**Comment by menovak** [comment]
hi mraccess77,

Tested this in FF 74 and Chrome 80, using JAWS 2018, JAWS 2019 and JAWS 2020 and found same results - the 'submenu' was not announced. Parallel testing using NVDA and the 'submenu' was announced.  I will file as JAWS bug.

Thanks, 
Regards,
Mark

**Comment by BenKeyFSI** [comment]
This bug is in BugZilla as bug 106002 {Chrome: JAWS does not indicate "sub menu" on dropdown items in menu bars}. It was added to BugZilla on 2018-12-20.
